### PR TITLE
badCode

### PR DESCRIPTION
**Pull request by tobwiens**
badCode

### CODE REVIEW

File: scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/ProgressFileReader.java [hunk 1]
```diff
@@ -80,9 +80,9 @@ boolean start(File workingDir, String filename) {
 
             return true;
         } catch (IOException e) {
```
Comment (sonarqube-activeeon) [inline scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/ProgressFileReader.java]: MAJOR Either remove or fill this block of code. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00108)
CRITICAL Either log or rethrow this exception. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 2 issues:
* CRITICAL 1 critical
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 2 issues:
* CRITICAL 1 critical
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by tobwiens** [comment]
That worked nice! Sonarqube commands are now working.

**Pull request closed without merging**